include/boost/config/compiler/sunpro_cc.hpp

```diff
@@ -203,8 +203,8 @@
 #endif
 //
 // last known and checked version is 0x590:
```

Comment:
The comment says the last supported version is "0x590" whereas the new code adjusts this to 0x5150. Maybe just omit the version from the comment or delete it?